docs/variables.md

```diff
@@ -155,6 +155,18 @@ returns `0`. Please review the Sprig documentation to understand which functions
 | `tasks.<TASKNAME>.outputs.parameters.<NAME>` | Output parameter of any previous task. When the previous task uses 'withItems' or 'withParams', this contains a JSON array of the output parameter values of each invocation |
 | `tasks.<TASKNAME>.outputs.artifacts.<NAME>` | Output artifact of any previous task |
 
+### RetryStrategy
+
+When using the `when` field within `retryStrategy`, special variables are available.
+
+| Variable | Description|
+|----------|------------|
+| `lastRetry.exitCode` | Exit code of the last retry |
+| `lastRetry.Status` | Status of the last retry |
+| `lastRetry.Duration` | Duration in seconds of the last retry |
```

Comment:
Variable name cannot be prefaced with `retries.` because that's already in use by the `{{retries}}` variable. In the enhanced expression, `retries` is of type string so it cannot have child members (eg `retries.last`)